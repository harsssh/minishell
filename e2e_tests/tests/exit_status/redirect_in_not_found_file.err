< zettai_sonnzai_shinai_file