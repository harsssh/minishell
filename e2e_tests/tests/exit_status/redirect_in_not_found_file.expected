minishell: zettai_sonnzai_shinai_file: No such file or directory
